### ISSUE TITLE
(unity 2018.1.0f2) Point lights do not express in LWRP.

### ISSUE THREAD

**Issue by unity3d-kr** [Unity-Technologies/ScriptableRenderPipeline#1322]
test:
1. new scene
2. add sphere(or other primitive), set LightweightShader.shader
3. add point light -> no expression. bug!
4. add one more point light -> expression.

I fix it for you.
https://github.com/unity3d-kr/ScriptableRenderPipeline

Download and overwrite LightweightPipeline.cs file.

**Comment by equalsequals** [comment]
Can you elaborate on what you've changed? I did a diff of your file versus the latest, but it appears I am branched at a newer revision.

Was it just this addition?
```
if (m_LocalLightIndices.Count >= m_MaxLocalLightsShadedPerPass)
                    break;
```

**Comment by unity3d-kr** [comment]
https://github.com/unity3d-kr/ScriptableRenderPipeline/commit/942e08362e6a2dc475c457544c15ffac9c771d10#diff-893b05e2b1a2b667e4d92edf4ee3a661

**Comment by phi-lira** [comment]
Hey @unity3d-kr. That tag has a weird in between state of LWRP. The issue is either fixed on master or LWRP but I can't remember just now which. LWRP will be merged soon to master. 

Be aware that we are refactoring the LWRP to make it more modular easily extendable. So, expect some heavy changes on the LWRP pipeline.

**Comment by phi-lira** [comment]
Fixed in #1216
